### PR TITLE
Add security and trust seals to jetpack checkout

### PR DESCRIPTION
**Pull request by CodeyGuyDylan**
Adds security and trust seals to Jetpack Checkout

## Proposed Changes

* Update the money-back guarantee icon
* Add a Security section below the 7/14 day money-back guarantee

P2: pbNhbs-1c7-p2

## Testing Instructions

1. Checkout this branch or use the calypso live branch
2. Go to `/checkout/jetpack/jetpack_starter_yearly`
3. Ensure you see the security and trust seals below the pay button
image
4. Make sure it scales well to tablet and mobile sizes
5. Go to `/checkout/akismet/ak_personal_yearly:-q-36` and make sure the security and trust seals do not show up here.
image
6. Go to `/checkout/business-bundle` and choose a site. Make sure the seals do not show up here as well
image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by sirbrillig** [comment]
Just a note that the lock icon will conflict slightly with the checkout redesign (p9Jlb4-6FB-p2) where we're adding a credit card icon to the submit button (slow going but currently in-progress at https://github.com/Automattic/wp-calypso/pull/77353).  
Screenshot 2023-06-01 at 2 12 44 PM

We could certainly fork the icon so that Jetpack checkout uses the lock in place of the credit card but I just wanted to make sure these things were kept in mind!

**Comment by michaeldcain** [comment]
> We could certainly fork the icon so that Jetpack checkout uses the lock in place of the credit card but I just wanted to make sure these things were kept in mind!

I left a comment on the design post to get some clarification on the lock (pbNhbs-1c7-p2#comment-15911). I think the CC icon makes more sense. There's also the consideration that alternative payment methods (PayPal, Google, Apple, etc) all have their own submit button designs with their own icons.

**Comment by CodeyGuyDylan** [comment]
Thank you both for this context! I am more than happy to remove the lock, I will wait to get @simonktnga8c's feedback to confirm. I was not aware of the project in progress to add the CC icon, this does seem more intuitive

**Comment by simonktnga8c** [comment]
I just replied on the P2; please go ahead with not including the lock icon @CodeyGuyDylan - thanks!

File: client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx [hunk 1]
```diff
@@ -46,7 +47,6 @@ import useValidCheckoutBackUrl from 'calypso/my-sites/checkout/composite-checkou
 import { leaveCheckout } from 'calypso/my-sites/checkout/composite-checkout/lib/leave-checkout';
 import { prepareDomainContactValidationRequest } from 'calypso/my-sites/checkout/composite-checkout/types/wpcom-store-state';
 import useCartKey from 'calypso/my-sites/checkout/use-cart-key';
-import { useDispatch as useReduxDispatch, useSelector } from 'calypso/state';
```
Comment (sirbrillig) [inline client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx]: I think we actually want to use these from `calypso/state` to preserve Type safety as per this request: p4TIVU-azW-p2

Comment (CodeyGuyDylan) [reply client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx]: Oh ok, this was done when I rebased, I didn't realize that difference. I'll get that fixed up!

**Review by elliottprogrammer — COMMENTED** [review]
Tested this and everything looks good and works good per testing instructions, however I noticed an issue where when the right checkout column has very long content, I cannot scroll down enough to see the SSL seal. The SSL seal gets cut off and I cannot scroll down further. See video screencast:

https://github.com/Automattic/wp-calypso/assets/11078128/96e0f994-c96e-4720-a928-2206043d0324

**Comment by CodeyGuyDylan** [comment]
> Tested this and everything looks good and works good per testing instructions, however I noticed an issue where when the right checkout column has very long content, I cannot scroll down enough to see the SSL seal. The SSL seal gets cut off and I cannot scroll down further. See video screencast:

Thank you for catching this, this is an interesting issue 🤔 I am not sure there is an easy fix. @simonktnga8c it looks like the seals are cut off when a 2 year plan is available. I could enable scrolling to make sure the user can scroll through the whole thing, but I don't think it's great to have the checkout scroll and the sidebar scroll.

Should we perhaps switch to the design where these seals are below the checkout button?

**Comment by sirbrillig** [comment]
> I could enable scrolling to make sure the user can scroll through the whole thing, but I don't think it's great to have the checkout scroll and the sidebar scroll.

Yeah, this has been an issue before and the new checkout design (p9Jlb4-6FB-p2) takes a different approach for the sidebar. For now I'd suggest not putting anything new in there if possible.

**Comment by simonktnga8c** [comment]
Sounds like the bottom is the best place so Dylan 👍 cheers

**Review by elliottprogrammer — APPROVED** [review]
Tested and works good and looks good, as described. Code looks all good.
LGTM 👍

**Comment by CodeyGuyDylan** [comment]
> Tested and works good and looks good, as described. Code looks all good. LGTM 👍

Thanks for the reviews and for catching that scrolling issue!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8029913

Thank you @CodeyGuyDylan for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8029913).

**Pull request merged**